### PR TITLE
Filterbox style improvements

### PR DESCRIPTION
**Pull request by snurrfint**

### CODE REVIEW

**Comment by hrigner** [comment]
Looks great! The selection filterbox is a bit too wide though.. 
screen shot 2018-11-07 at 07 28 45

**Comment by snurrfint** [comment]
> Looks great! The selection filterbox is a bit too wide though..
> screen shot 2018-11-07 at 07 28 45

Nice catch, didn't notice!

File: src/components/filterbox.pcss [hunk 1]
```diff
@@ -2,12 +2,26 @@
...
   box-sizing: border-box;
   color: #333;
-  font-size: 1.5rem;
+  font-size: 1rem;
   height: 100%;
   outline: none;
   position: relative;
   user-select: none;
 
+  *::-webkit-scrollbar {
```
Comment (peol) [inline src/components/filterbox.pcss]: Not too fond of us fiddling with scrollbar styles (which are affected by e.g. theming/accessibility settings in their OS).

File: src/components/filterbox.pcss [hunk 2]
```diff
@@ -28,17 +42,23 @@
 
   .search {
     background: none;
-    border: 0;
-    border-top: 1px solid rgb(201, 201, 201);
+    border: 1px solid #d9d9d9;
     box-sizing: border-box;
     display: inline-block;
+    font-family: 'Source sans pro', sans-serif;
```
Comment (peol) [inline src/components/filterbox.pcss]: Could we move this declaration to `index.pcss` and have it for `input, textarea` instead?

File: src/components/filterbox.pcss [hunk 3]
```diff
@@ -28,17 +42,23 @@
...
     font-size: 1.5rem;
     height: 2em;
     left: 0;
     outline: none;
+    padding: 0.5em;
```
Comment (peol) [inline src/components/filterbox.pcss]: We're mixing em and rem, I think we should stick to one (em's are a PITA to work with in my opinion :))

File: src/components/filterbox.pcss (hunk 2 above)
Comment (snurrfint) [reply src/components/filterbox.pcss]: sounds good.

File: src/components/filterbox.pcss (hunk 1 above)
Comment (snurrfint) [reply src/components/filterbox.pcss]: Sure. I'll remove this.

File: src/components/filterbox.pcss (hunk 3 above)
Comment (snurrfint) [reply src/components/filterbox.pcss]: Ah, It's not PITA to work with. However, I was doing it wrong in this case. Thanks for pointing that out.

We should make sure we are consistent with the use of rem em in the project at some point. If it's too messy we could just use rems everywhere I guess.

**Pull request merged**